test/language/expressions/arrow-function/arrow/bind_0.js

```diff
@@ -0,0 +1,12 @@
+// Copyright 2015 Microsoft Corporation. All rights reserved.
+// This code is governed by the license found in the LICENSE file.
+
+/*---
+description: invoke arrow function using bind
+es6id: 14.2
+---*/
+
+var a = (x) => x;
+
+assert.sameValue(typeof a.bind, "function", "Bind is not a member exression on arrow function (x) => x.");
+assert.sameValue(a.bind.length, 1, "Lengh property of bind method should be 1.");

```

Comment:
Duplicates 

- https://github.com/tc39/test262/blob/master/test/language/expressions/arrow-function/lexical-this.js
- https://github.com/tc39/test262/blob/master/test/language/expressions/arrow-function/prototype-rules.js